ui/browser/reducers/main-reducers.js

```diff
@@ -52,6 +53,9 @@ export default function basic(state = initialState, action) {
     case types.SET_PAGE_ORDER:
       return setPageOrder(state, action.pageOrder);
 
+    case types.TOGGLE_PAGES_SECTION:
+      return togglePagesSection(state, action.opened);
```

Comment:
I really don't like the past tense "opened" here -- it's unclear what the future state will be, as the actions cause a change in state via reducer, and seems to imply it's referring to the current state, and the 'toggle' reference could imply the inverse. Either way, I think as a whole, this action/state should be closer to `SET_PAGES_VIEW_VISIBILITY` or something, where it doesn't reference a previous state (which 'toggle' and a past tense verb seem to imply).